### ISSUE TITLE
[Bug] Dynamic Island activating when still within YouTube app

### ISSUE THREAD

**Issue by Blarnix** [arichorn/uYouPlusExtra#69]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Select uYou Label

- [ ] uYou

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Device info

- iOS/iPadOS version: 16.6
- Device model: iPhone 14 Pro Max
- Sideload tool (AltStore, Sideloadly, TrollStore,...): AppDB
- The specific version of uYouPlus (**latest or newest is NOT a version number!**): 18.29.1-3.0.1

### Describe the bug

[Attached](https://share.icloud.com/photos/07bpQYztH5lni2fusC4bQUyiQ)  is what I mean by the bug, when I go into any video, the dynamic island activates with the audio controls while still within the YouTube app. This just suddenly started happening and I have no idea why, I wanted to see if anyone else was aware.

### Steps to reproduce the issue

1. Open YouTube
2. Play a video 
3. Dynamic island activates

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

**Comment by Blarnix** [comment]
Yeah it's really, really weird. I only began to notice it when I began using AppDB and I'm suspecting it has something to do with the NowPlaying API that might not be called correctly. The other odd thing about it is that whenever I click on the dynamic island, it **doesn't** open YouTube, it's like it doesn't exist. Furthermore, when playing the audio, ![the YouTube logo is missing from control center.](https://github.com/arichorn/uYouPlusExtra/assets/61402697/e4f1520a-95d6-4182-b3e3-5d846e053440) I have absolutely no idea why, but I suspect some kind of bundle/group ID issue? No clue.

**Comment by KieranBLK** [comment]
> Yeah it's really, really weird. I only began to notice it when I began using AppDB and I'm suspecting it has something to do with the NowPlaying API that might not be called correctly. The other odd thing about it is that whenever I click on the dynamic island, it **doesn't** open YouTube, it's like it doesn't exist. Furthermore, when playing the audio, the YouTube logo is missing from control center.  ![the YouTube logo is missing from control center.](https://github.com/arichorn/uYouPlusExtra/assets/61402697/e4f1520a-95d6-4182-b3e3-5d846e053440) I have absolutely no idea why, but I suspect some kind of bundle/group ID issue? No clue.

These are due to a lack of entitlements in the IPA. Dynamic island and spatial audio are both entitlements that can't be generated by enterprise certificates. You'll need to side load with your Apple ID in order to generate those entitlements.

**Comment by Blarnix** [comment]
﻿Sadly, that didn’t work.

**Comment by Youwekk** [comment]
This stopped happening for me after I disabled YouPip and disabled "Background Playback" option in uYou settings > Other Settings
But I lost ability to watch in background and Picture in Picture :(

**Comment by hi-im-karma** [comment]
Im a YouTube Premium member and have this issue with both uyou+/enhanced. I disabled both these options and the Dynamic Island is still showing...![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/290dff31-3062-4d15-a6e7-c69b3ae5c2cb)
![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/28bcb4e9-3b84-45f0-b692-f1055335dfc1)

**Comment by lxly9** [comment]
yeah, seems that this only happens when uyouenhanced is using the youtube bundle id, as it doesnt happen when the app is sideloaded via sideloadly or sidestore.

**Comment by aricloverEXTRA** [comment]
Hey @lxly9 @Blarnix @KarmaBeast @KieranBLK 

Try out the newest release **21.14.4-3.0.5** I did some big changes related to Dynamic Island.
